apps/etterlatte-behandling/src/main/resources/db/migration/V165__trigger_lytte_endringer.sql

```diff
@@ -0,0 +1,49 @@
+-- Basert på https://www.cybertec-postgresql.com/en/tracking-changes-in-postgresql/
+CREATE SCHEMA logging;
```

Comment:
Burde vi vurdere å droppe eget skjema for dette inntil det blir bedre støttet i nais? Pr nå sliter vi vel litt med tilgang til det vi legger på egne skjema - ref problemene @sebassonav har vært borti